wallet/index.js

```diff
@@ -73,42 +75,45 @@ class Wallet {
     constructor(ctx) {
         this.ctx = ctx;
         this.log = ctx.log.child({module: 'Wallet'});
-        this.config = ctx.config.client.wallet;
-
+        this.keystorePath = path.join(config.get('datadir'), config.get('wallet.keystore_path'));
         // Events
         // transactionEventEmitter emits the TRANSACTION_EVENT type
         this.transactionEventEmitter = new events.EventEmitter();
     }
 
     async start() {
-        this.keystore_path = path.join(this.ctx.datadir, this.config.keystore_path);
-        if (!fs.existsSync(this.keystore_path)) {
-            fs.mkdirSync(this.keystore_path, {recursive: true});
+        await makeSurePathExistsAsync(this.keystorePath);
+        try {
+            this.#secretPhrase = JSON.parse(
+                await fs.promises.readFile(path.join(this.keystorePath, 'key.json'), 'utf-8')
+            ).phrase;
+        } catch (e) {
+            if (e.code === 'ENOENT') {
+                this.#secretPhrase = '';
```

Comment:
Why should we handle this case? Running the app without the phrase makes no sense.

Reply:
App is often run without a phraze and it was quite a long time ago, I was checking it. It was never created in docker before starting the app